### PR TITLE
refactor(planning_launch): remove minimum lane changing length

### PR DESCRIPTION
**Pull request by purewater0901**
## Description
Remove `lane changing length` parameter from the planning launch parameter file.

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/rviz/autoware.rviz [hunk 1]
```diff
@@ -18,10 +18,6 @@ Panels:
     Splitter Ratio: 0.5
   - Class: tier4_localization_rviz_plugin/InitialPoseButtonPanel
     Name: InitialPoseButtonPanel
-  - Class: AutowareDateTimePanel
-    Name: AutowareDateTimePanel
-  - Class: rviz_plugins::AutowareStatePanel
-    Name: AutowareStatePanel
```
Comment (rej55) [inline autoware_launch/rviz/autoware.rviz:24]: Unnecessary change is included.
Could you check?

Comment (purewater0901) [reply autoware_launch/rviz/autoware.rviz:24]: Sorry, I'll fix it

Comment (purewater0901) [reply autoware_launch/rviz/autoware.rviz]: @rej55 
Fix https://github.com/autowarefoundation/autoware_launch/pull/308/commits/447b174a2f17fa911fb1fd9e98717cb283532eed

**Review by rej55 — APPROVED** [review]
